camera_control/camera_control/camera_node_client.py

```diff
@@ -0,0 +1,65 @@
+import rclpy
+
+from std_srvs.srv import Empty
+from camera_control_interfaces.srv import GetBool, GetPosData
+from laser_control_interfaces.msg import Point
+
+
+# Could make a mixin if desired
+class CameraNodeClient:
+    def __init__(self, node):
+        node.camera_get_lasers = node.create_client(
+            GetPosData, "camera_control/get_laser_detection"
```

Comment:
nit: we could pull these service names out as constants so it's harder to break via a typo

Reply:
I was thinking of adding something like this if we wanted to support multiple nodes as well. The name would become like {service}_{idx}/function_name